src/layouts/Tutorial.tsx

```diff
@@ -223,21 +223,25 @@ export const TutorialLayout = ({
           />
           {children}
           {frontmatter.lang !== "en" ? (
-            <Text>Crowdin contributor</Text>
+            // TODO: Implement CrowdinContributors
+            <>
+              <Text>Crowdin contributor</Text>
```

Comment:
I don't see this text in prod, I think it was left as a reference. We should remove it